code/modules/clothing/under/ties.dm

```diff
@@ -727,18 +630,17 @@ obj/item/storage/internal/accessory/knifeharness/duelling
 	var/drawSound = 'sound/weapons/gun_pistol_draw.ogg'
 	storage_flags = STORAGE_ALLOW_QUICKDRAW|STORAGE_FLAGS_POUCH
 	can_hold = list(
-		/obj/item/weapon/gun/pistol,
-		/obj/item/ammo_magazine/pistol,
-		/obj/item/ammo_magazine/pistol/heavy,
-		/obj/item/ammo_magazine/pistol/heavy/super,
-		/obj/item/ammo_magazine/pistol/heavy/super/highimpact,
-		/obj/item/weapon/gun/revolver/m44,
-		/obj/item/ammo_magazine/revolver,
-		/obj/item/weapon/gun/pistol/smart,
-		/obj/item/ammo_magazine/pistol/smart,
-		/obj/item/weapon/gun/energy/taser,
-		/obj/item/weapon/gun/flare
-	)
+
+//Can hold variety of pistols and revolvers together with ammo for them. Can also hold the flare pistol and signal/illumination flares.
+	/obj/item/weapon/gun/pistol,
+	/obj/item/weapon/gun/energy/taser,
+	/obj/item/weapon/gun/revolver,
+	/obj/item/ammo_magazine/pistol,
+	/obj/item/ammo_magazine/revolver,
+	/obj/item/ammo_magazine/internal/flare,
+	/obj/item/device/flashlight/flare
+
```

Comment:
```suggestion
	/obj/item/device/flashlight/flare,
	/obj/item/weapon/gun/flare
```
needs that flaregun there

Reply:
Added the actual flaregun into the hold list